src/main/g8/client/src/main/scala/ClientApp.scala

```diff
@@ -0,0 +1,26 @@
+package $package$.client
+
+import cats.effect._
+import fs2.Stream
+import io.chrisdavenport.log4cats.slf4j.Slf4jLogger
+import monix.execution.Scheduler
+
+object ClientApp {
+
+  implicit val S: Scheduler = monix.execution.Scheduler.Implicits.global
+
+  implicit val TM = Timer.derive(Effect[IO], IO.timer(S))
+
+  def main(args: Array[String]): Unit = {
+    (for {
+      logger <- Stream.eval(Slf4jLogger.fromName[IO]("Client"))
+      client <- {
+        implicit val l = logger
+        PeopleServiceClient.createClient[IO]("localhost", 19683)
+      }
+      getPersonResponse <- Stream.eval(client.getPerson("foo")).as(println)
+    } yield (getPersonResponse)).compile.toVector.unsafeRunSync()
```

Comment:
I feel like we don't need to create an `fs2.Stream` here and then evaluate it immediately. What do you think?

Reply:
Initially (a based on the Adri's example) the client boot was a fully StreamApp, but I added some changes but I have kept how the RPC client is created (via `ClientCache`) and this is a constraint at the moment.